indoor-floor-level-tracker/firmware/src/main.cpp

```diff
@@ -74,6 +74,14 @@ void setup()
   notecard.begin();
 
 #ifdef USE_DISPLAY
+#ifdef DISPLAY_POWER_PIN
+  pinMode(DISPLAY_POWER_PIN, OUTPUT);
+  digitalWrite(DISPLAY_POWER_PIN, HIGH);
+#endif
+#ifdef DISPLAY_GROUND_PIN
+  pinMode(DISPLAY_GROUND_PIN, OUTPUT);
+  digitalWrite(DISPLAY_GROUND_PIN, LOW);
+#endif
```

Comment:
Dang, good catch. Sorry I thought I looked this over but I guess I missed this chunk too 🙁 

I’m still not sure how this happened. That code is in the [code in `main` before the merge](https://github.com/blues/app-accelerators/blob/9c47bd18ff38349eefbce0cd6f531585a3de6bc3/indoor-floor-level-tracker/firmware/src/main.cpp). Mat did comment out the display in [this commit](https://github.com/blues/app-accelerators/pull/8/commits/1ffe30083ed93e69542d53066b0775b30a1bd790), but that’s it.

So I can’t for the life of me figure out why 6c0a4824a4fc8f12b73dd4430f69c808c840c99d doesn’t include these `define`s.

For now I put them back in, tweaked the comments, and made sure everything is still building ok. Since the display is working for you now I’m hoping we’re good. Sorry for the mess here.